recipes-overlayed/networkmanager/networkmanager_%.bbappend

```diff
@@ -0,0 +1,17 @@
+do_install_append_qemuarm64 () {
```

Comment:
why would meta-rpb be the 'right' place for this change? shouldn't that go with networkmanager recipe directly?